e2e_test/streaming/extreme_null.slt

```diff
@@ -36,8 +36,8 @@ query RRRR
 select sum_v1, min_v1, max_v1, v3 from mv3 order by sum_v1;
 ----
 0 0 0 3
-1 NULL 1 4
-2 NULL 2 1
+1 1 NULL 4
+2 2 NULL 1
```

Comment:
Why change this?

Reply:
Because now in our system we assume `NULL` the largest, just like PG. But this actually still not correct for min/max, because `NULL` is filtered out in min/max agg. I may solve this inconsistency to PG in later PR, or open an issue to record it.